server/api/server.go

```diff
@@ -26,9 +26,9 @@ import (
 
 // Server defines the server's API interface.
 type Server struct {
-	Router     *mux.Router
-	GRPCRouter *runtime.ServeMux
-	ClientCtx  client.Context
+	Router            *mux.Router
```

Comment:
While we're at it, we could also rename this one to LegacyAPIRouter

Reply:
I think we can rename this to `LegacyRouter`. There already seems to be a `GRPCGatewayRouter` for the gateway.

Reply:
Yes, it's the main router, we cannot rename it as Legacy